### DISCUSSION TITLE
Translation messages from a different source

### DISCUSSION THREAD

**Discussion by LudwigNel** [nuxt-modules/i18n#1375] [Q&A]
Is there a way to use a different source to load the translation messages?
I tried using axios get in the i18n config but that never seems to call the api I want to use to supply the messages

**Comment by rchl** [comment]
The `file` property on the locale object specified in nuxt.config can point to a javascript file that exports an async function. More info in https://i18n.nuxtjs.org/lazy-load-translations.
